llvm/tools/sycl-post-link/sycl-post-link.cpp

```diff
@@ -306,9 +311,19 @@ std::string saveModuleIR(Module &M, int I, StringRef Suff) {
 
 std::string saveModuleProperties(module_split::ModuleDesc &MD,
                                  const GlobalBinImageProps &GlobProps, int I,
-                                 StringRef Suff, StringRef Target = "") {
-  auto PropSet =
-      computeModuleProperties(MD.getModule(), MD.entries(), GlobProps);
+                                 StringRef Suff, StringRef Target = "",
+                                 bool IsBF16DeviceLib = false) {
```

Comment:
At first glance passing this flag through several APIs seems a little bit too intrusive, especially if we are going to have more "special" libraries like this. However, I don't have any immediate suggestions on how to generalize this, so the comment can be ignored for now, consider it just me thinking out loud.

Reply:
Hi, @AlexeySachkov 
I don't like adding an extra parameter for these functions in sycl-post-link either, so updated the pr to avoid breaking previous API prototype for functions such as "saveModule", "computeModuleProperties".
Thanks very much.